src/docs/maturity/enterprise-scale.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Enterprise Scale
+description: Scale to trillions of requests with Railway Enterprise with ease. Enhanced security, premium support, and custom deployment options.
+---
+
+Railway Enterprise takes scale to a whole new level. The Enterprise tier delivers the reliability and compliance you need, with the experience your developers deserve. Built for organizations that require enterprise-grade security, dedicated resources, and comprehensive control over their infrastructure.
+
+## Performance at Scale
+
+Railway accommodates the smallest startups to the biggest enterprises, used among 23% of the Fortune 500 companies.
+
+At scale Railway processes:
+- 15+ million monthly deployments
+- 1+ trillion edge network requests
+
+The platform supports your production-grade workloads with:
+- 1TB RAM memory limits
+- 1000 vCPU processor limits
+- 50TB disk storage limits
+- 99.999% SLA
+- 50ms p95 global network RTT
+
+## Enterprise Features
+
+### Security and Compliance
+
+Railway Enterprise supports your compliance requirements with industry-standard certifications including SOC 2 Type II and HIPAA compliance. For more detailed information, see [our compliance documentation](https://docs.railway.com/maturity/compliance).
+
+Authentication on Railway:
+- Single Sign-On (SSO)
+- SAML authentication
+- Granular access control
+- Audit logs, including change tracking
+
+All of Railway runs with zero-trust networking:
+- End-to-end encryption
+- Private networking between services
+- No manual firewall configuration needed
+
+### Deployment Options
+
+Railway can be deployed globally, with availability across the Americas, EMEA, and APAC.
+
+**Railway-managed infrastructure**: This option lets you get started quickly, while continuing to stay secure. Our standard cloud offering can scale to thousands of requests per second seamlessly.
+
+**Railway-managed dedicated VMs**: Your workloads run on dedicated Railway hosts for complete resource isolation. Continue to benefit from Railway-managed infrastructure with added security.
+
+**Bring your own cloud**: Deploy Railway within your VPC for ultimate compliance within your infrastructure. By using your existing cloud account, consume credits or use existing commercial commitments and EDP minimums while experiencing Railway's world-class developer experience.
+
+### Premium Support
+
+On the enterprise tier, receive prioritized support from our world-class engineering team, with:
+- Private Slack channel
+- 24/7 support coverage
+- 1hr guaranteed and 5 minute median response times on critical issues
```

Comment:
- should read `1 hour` SLO

`median response times` should read `MttR` (mean time to respond) as the standard parlance. (Standard terms here are MttA, MttR, TTR --- our current MttR is 1h 34m)

All that said maybe just leave this bulletpoint off? MttRs are internal metrics, SLOs are external.

Reply:
this was a rahul suggestion, asking in discord